api/src/pcapi/scripts/remove_pro_from_beneficiary/main.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+import logging
+
+from pcapi.core.offerers import models as offerers_models
+from pcapi.core.users import models as users_models
+from pcapi.flask_app import app
+from pcapi.models import db
+from pcapi.models.validation_status_mixin import ValidationStatus
+
+
+logger = logging.getLogger(__name__)
+
+
+def main(args: argparse.Namespace) -> None:
+    user = users_models.User.query.filter(users_models.User.id == args.user_id).one()
+    if not (user.has_beneficiary_role or user.has_underage_beneficiary_role):
+        logger.error("Cannot remove pro roles from beneficiary %s. They have no BENEFICIARY roles", args.user_id)
+        return
+    if not (user.has_beneficiary_role or user.has_non_attached_pro_role):
+        logger.error("Cannot remove pro roles from beneficiary %s. They have no PRO roles", args.user_id)
+        return
+    user_offerer_count = offerers_models.UserOfferer.filter(
+        offerers_models.UserOfferer.userId == args.user_id,
+        offerers_models.UserOfferer.validationStatus == ValidationStatus.VALIDATED,
```

Comment:
Ne voudrais-tu pas plutôt explicitement non rejeté ?
```suggestion
        ~offerers_models.UserOfferer.isRejected,
```